jemoji.gemspec

```diff
@@ -12,7 +12,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'jekyll', '>= 3.0'
   s.add_dependency 'html-pipeline', '~> 2.2'
   s.add_dependency 'activesupport', '~> 4.0'
-  s.add_dependency 'gemoji', '~> 2.0'
+  s.add_dependency 'gemoji', '~> 3.0'
 
```

Comment:
I can't comment on whether this breaks any backward compatibility. Is Gemoji 3.0 compatible with still using GitHub hosting of assets?